components/supervisor/pkg/terminal/service.go

```diff
@@ -21,10 +22,18 @@ import (
 
 // NewMuxTerminalService creates a new terminal service
 func NewMuxTerminalService(m *Mux) *MuxTerminalService {
+	shell := os.Getenv("SHELL")
+	if shell == "/bin/false" {
```

Comment:
Who sets shell to `/bin/false`, i.e. why is this a case that we need to handle?
Maybe a comment that clarifies this case would be handy.

Reply:
I copied it from VS Code but I will revert it till we have a real issue